pre-miss/ArticleAPI.js

```diff
@@ -0,0 +1,80 @@
+import axios from 'axios';
+
+// ✅ Axios 인스턴스 생성
+const instance = axios.create({
+    baseURL: "https://panda-market-api-crud.vercel.app/articles",  
+    timeout: 10000,
+});

```

Comment:
```suggestion
const pandaMarketApi = axios.create({
    baseURL: "https://panda-market-api-crud.vercel.app/articles",  
    timeout: 10000,
});

```
instance라는 변수명은 어떤 역할을 수행하는지 특정하기 어려워 보입니다.
axios를 이용해서 panda-market api를 이용하는 instance를 만들었으므로 위와 같은 변수명으로 봐꿔서 사용하는 것이 더 좋을 것 같습니다.